react/src/objects/stack/SprkStack.stories.js

```diff
@@ -110,11 +106,11 @@ stackSplitFourCol.story = {
 
 export const stackSplitFiveCol = () => (
   <SprkStack splitAt="tiny">
-    <SprkStackItem additionalClasses="sprk-o-Stack__item--fifth@xs"></SprkStackItem>
-    <SprkStackItem additionalClasses="sprk-o-Stack__item--fifth@xs"></SprkStackItem>
-    <SprkStackItem additionalClasses="sprk-o-Stack__item--fifth@xs"></SprkStackItem>
-    <SprkStackItem additionalClasses="sprk-o-Stack__item--fifth@xs"></SprkStackItem>
-    <SprkStackItem additionalClasses="sprk-o-Stack__item--fifth@xs"></SprkStackItem>
+    <SprkStackItem additionalClasses="sprk-o-Stack__item--fifth@xs" />
```

Comment:
@bobwise do you think removing the closing tag makes it more difficult for users of Spark?

Reply:
@afebbraro This is the only way I could get it under 80 characters. If I put it on a new line Prettier undoes it automatically. I think this is definitely more difficult for Spark users.

Reply:
@bobwise ah okay. In the case where linting hurts usability for our consumers, lets disable it for that line or lines.